docs/dev/rfc/odfe-cli-rest-api-as-commands.md

```diff
@@ -0,0 +1,298 @@
+# RFC: Support REST api as command
+
+## Objective
+
+This document describes how to support [rest api](https://www.elastic.co/guide/en/elasticsearch/reference/current/rest-apis.html) in a generic way.
+As of now, elasticsearch support api in different categories like index api, cat api, cluster api, document api, etc to
+configure and access their features. It will take tremendous amount of time and effort to provide every api as native commands.
+Hence, we will be categorizing those REST API based on GET/PUT/POST/DELETE  and allow users to perform their request 
```

Comment:
We should be consistent in capitalization of "rest" and "api"

Reply:
Ack

Reply:
I still see some incosistencies where there is "api" and "API"